lib/stdlib/src/lists.erl

```diff
@@ -1302,8 +1236,7 @@ performed on the `N`th element of the tuples. The sort is stable.
 -spec keysort(N, TupleList1) -> TupleList2 when
       N :: pos_integer(),
       TupleList1 :: [Tuple],
-      TupleList2 :: [Tuple],
-      Tuple :: tuple().
+      TupleList2 :: [Tuple].
```

Comment:
I think `keysort/2` and all the keymerge variants do require a list of only tuples.

```erl
> 13> lists:keymerge(1, [a,{a,b}],[a,{a,b}]).
** exception error: bad argument
     in function  element/2
        called as element(1,a)
        *** argument 2: not a tuple
     in call from lists:keymerge_1/3 (lists.erl, line 1384)
```

Reply:
Thanks for catching this, I wasn't aware. I've reverted the changes to these specs.